internal/endtoend/testdata/copyfrom_imports/postgresql/pgx/query.sql

```diff
@@ -6,3 +6,7 @@ INSERT INTO myschema.foo (a, b) VALUES ($1, $2);
 
 -- name: InsertSingleValue :exec
 INSERT INTO myschema.foo (a) VALUES ($1);
+
+-- name: DeleteValues :execresult
```

Comment:
This was the main offender, any file generated from a query file with queries of type `metadata.CmdExecResult` would include the `github.com/jackc/pgconn` package